documents/Libraries/stdlib/sx-glsl/mx_gamma22_to_linear_color3.glsl

```diff
@@ -0,0 +1,5 @@
+void mx_gamma22_to_linear_color3(vec3 _in, out vec3 result)
+{
+    vec3 gamma = vec3(2.200000047683716, 2.200000047683716, 2.200000047683716);
```

Comment:
One thought here:

Although these values will drift from 2.2 in real-world implementations (due to floating point precision), the underlying constant that we're trying to represent is exactly 2.2.  Would it make sense to write the exact constant here, and let the graphics card approximate it as needed for its own internal representation?

Reply:
After speaking with one of our color management experts, it seems that you are correct, I'll make the changes.